src/manifold/src/boolean3.cpp

```diff
@@ -92,9 +92,12 @@ SparseIndices Filter11(const Manifold::Impl &inP, const Manifold::Impl &inQ,
                        const SparseIndices &p1q2, const SparseIndices &p2q1) {
   ZoneScoped;
   SparseIndices p1q1(3 * p1q2.size() + 3 * p2q1.size());
-  for_each_n(autoPolicy(p1q2.size()), zip(countAt(0), countAt(0)), p1q2.size(),
-             CopyFaceEdges<false>({p1q2, p1q1, inQ.halfedge_}));
-  for_each_n(autoPolicy(p2q1.size()), zip(countAt(p1q2.size()), countAt(0)),
+  for_each_n(
+      autoPolicy(p1q2.size()),
+      zip(countAt(static_cast<size_t>(0)), countAt(static_cast<size_t>(0))),
```

Comment:
Yes if we want to change everything into using `size_t`. Thinking about it, we can also use `0ll` to replace the `static_cast`. I used to use `0l` but it errors on Windows, because `long` is 32-bits on Windows and we need `long long`, so I just switched everything into `static_cast`...

Reply:
ah, nevermind, https://en.cppreference.com/w/cpp/language/integer_literal size_t suffix is only supported on c++23, we can use something user defined though.

Reply:
now we can use `0_z` where the `_z` suffix specifies the number is a `size_t`.